xicam/gui/bluesky/search.py

```diff
@@ -47,11 +47,19 @@
 RELOAD_INTERVAL = 11
 _validate = functools.partial(jsonschema.validate, types={'array': (list, tuple)})
 
+def timeit(f):
+    def wrap(*args):
+        time1 = time.time()
+        ret = f(*args)
+        time2 = time.time()
+        print('{:s} function took {:.3f} ms'.format(f.__name__, (time2-time1)*1000.0))
+        return ret
+    return wrap
+
 
 def default_search_result_row(entry):
-    metadata = entry.describe()['metadata']
-    start = metadata['start']
-    stop = metadata['stop']
+    start = entry.metadata['start']
+    stop = entry.metadata['stop']
```

Comment:
This changed help the execution time of show_results. For a set of about 10 runs it took about 2.2s before and ~1ms after. describe() calls BlueskyRun._load(), which is not needed here because the run has already been loaded. BlueskyRun._load() takes a while to execute, so I tried to minimuze the number of times it is called.

Reply:
Now that we have a solution for the serialization issue, is this change still impactful? I recall that we adopted `describe()` intentionally here, though I'm unfortunately fuzzy on the reasoning.